src/main/scala/com/fulcrumgenomics/commons/io/PathUtil.scala

```diff
@@ -42,9 +44,12 @@ object PathUtil {
     */
   def sanitizeFileName(fileName: String,
                        illegalCharacters: String = PathUtil.illegalCharacters,
-                       replacement: Option[Char] = Some('_')): String = replacement match {
-    case None    => fileName.filter(c => !illegalCharacters.contains(c))
-    case Some(r) => fileName.map(c => if (illegalCharacters.contains(c)) r else c)
+                       replacement: Option[Char] = Some('_')): String = {
+    val sanitizedFileName = replacement match {
+      case None => fileName.filter(c => !illegalCharacters.contains(c))
+      case Some(r) => fileName.map(c => if (illegalCharacters.contains(c)) r else c)
```

Comment:
Don't get rid of the precious `=>` alignment!
```suggestion
      case None    => fileName.filter(c => !illegalCharacters.contains(c))
      case Some(r) => fileName.map(c => if (illegalCharacters.contains(c)) r else c)
```